content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,291 @@
+The root namespace is configurable in the [`MeshConfig`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig)
+and has the default value of `istio-system`.
+
+For example, you installed Istio in `istio-system` namespace and deployed workloads in
+`default` and `bookinfo` namespace. The root namespace is configured to `istio-config`.
+The following policy will apply to workloads in every namespace including `default`,
+`bookinfo` and also the `istio-system`.
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: policy
+ namespace: istio-config
+spec:
+ rules:
+ # omitted
+{{< /text >}}
+
+### Ingress/Egress Gateway support
+
+The `v1beta1` authorization policy also supports to be applied on ingress/egress
+gateway to enforce access control on traffic entering/leaving the mesh, you just
+need to change the `selector` to make the policy to select the ingress/egress workload.
+
+The following policy applies to the workload with label `app: istio-ingressgateway` in
+`istio-system` namespace:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: ingress
+ namespace: istio-system
+spec:
+ selector:
+   matchLabels:
+     app: istio-ingressgateway
+ rules:
+ # omitted
+{{< /text >}}
+
+### Comparision
+
+The following table highlights the key differences between the old `v1alpha1` RBAC policies and
+the new `v1beta1` authorization policy.
+
+| Feature | `v1alpha1` RBAC policy | `v1beta1` Authorization Policy |
+|---------|------------------------|--------------------------------|
+| API stability | `alpha`: **No** backward compatible | `beta`: backward compatible **guaranteed** |
+| Number of CRDs | Three: `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding` | Only One: `AuthorizationPolicy` |
+| Mechanism to specify where to apply the policy | Specify by **service** | Specify by **workload** |
+| Deny-by-default behavior | Enabled **explicitly** by configuring `ClusterRbacConfig` | Enabled **implicitly** with `AuthorizationPolicy` |
+| Ingress/Egress gateway support | Not supported | Supported |
+| The `"*"` value in policy | Match all contents (empty and non-empty) | Match non-empty contents only |
+| The `destination.user` constraint | Supported | Not supported |
+
+Beyond all the differences, the `v1beta1` policy is enforced by the same engine in Envoy
+and supports the same authenticated identity (mutual TLS or JWT), condition and other
+primitives (e.g. IP, port and etc.) as the `v1alpha1` policy.
+
+See [authorization concept page](/docs/concepts/security/#authorization) for a detailed
+in-depth explanation of the `AuthorizationPolicy`.
+
+## Future of the `v1alpha1` policy
+
+With new `v1beta1` authorization policy, the `v1alpha1` RBAC policy (`ClusterRbacConfig`,
+`ServiceRole`, and `ServiceRoleBinding`) is deprecated and will not be supported from Istio 1.6 onwards.
```

Comment:
will be deprecated from Istio 1.6 onwards.

Reply:
Here I want to make it clear that the alpha API is deprecated in 1.4 and will be removed in 1.6. The `deprecate` is a signal that user should move away from it but we still support it to be used for some time. Updated to make this message more clear.